website/addons/figshare/templates/figshare_user_settings.mako

```diff
@@ -1,14 +1,15 @@
 <!-- Authorization -->
 <div>
     <h4 class="addon-title">
+      <img class="addon-icon" src="${addon_icon_url}"></img>
         figshare
         <small class="authorized-by">
             % if authorized:
-                    authorized
-                <a id="figshareDelKey" class="text-danger pull-right addon-auth">Delete Access Token</a>
+                    authorized by <a href="{{ profileUrl }}"><em> ${name} </em></a>
```

Comment:
It appears to incorrectly link to `http://localhost:5000/settings/addons/%7B%7B%20profileUrl%20%7D%7D`. I think using the explicit `data-bind` should fix this.